src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'Blue',
+}
+
+enum Shape {
+  Circle = 'circle',
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+}
 
+export interface Figure {
+  color: Color,
+  shape: Shape,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Circle implements Figure {
+  public shape = Shape.Circle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+  ) {
+    if (a <= 0) {
+      throw new Error('value can`t be 0 and <');
+    }
+  }
 
+  getArea(): number {
+    return Math.trunc((Math.PI * (this.a ** 2)) * 100) / 100;
+  }
```

Comment:
a method getArea that returns the area of the figure rounded down to hundredths.
Math.floor !== Math.trunc